plan.go

```diff
@@ -66,7 +67,7 @@ func Plan(ctx *PlanningContext) (*QueryPlan, error) {
 		return nil, fmt.Errorf("not implemented")
 	}
 
-	steps, err := createSteps(ctx, nil, parentType, "", ctx.Operation.SelectionSet, false)
+	steps, err := createSteps(ctx, nil, parentType, "", ctx.Operation.SelectionSet)
```

Comment:
I've eliminated the `childstep` recursion flag because its only purpose was to conditionally manage the inclusion of federation `id`s – to which I'm [not convinced it wasn't buggy](https://github.com/movio/bramble/pull/94/files#r744355229). Now that `_bramble_id` is ubiquitously included in all boundary scopes, the complexity of `childstep` is unnecessary.